### ISSUE TITLE
Custom js support

### ISSUE THREAD

**Issue by aleGuardiola** [JosephWoodward/graphiql-dotnet#46]
There is a lot of issues already that will need to modify the javascript of the library. Some of them are to generic for example there is any standard subscription protocol for graphql at this moment. So allowing some javascript configuration we can choose the fetch for subscriptios for example.

**Comment by josephwoodward** [comment]
I agree, this is something I considered as part of the previous release. Which files would be helpful to be able to modify? Or perhaps choose to provide your own one?

**Comment by aleGuardiola** [comment]
Honestly I am not sure, let me check again the current implementation and I can make a suggestion

**Comment by aleGuardiola** [comment]
Looking at the code we can pass the path of a custom config.js file where all the parameters of the graphiql component can be set in a global object. Exposing the default values that you already have implemented and using them as default.
